sonarr/languageprofile.go

```diff
@@ -0,0 +1,115 @@
+package sonarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"golift.io/starr"
+)
+
+// LanguageProfile is the /api/v3/languageprofile endpoint.
+type LanguageProfile struct {
+	UpgradeAllowed bool         `json:"upgradeAllowed"`
+	ID             int64        `json:"id,omitempty"`
+	Name           string       `json:"name"`
+	Cutoff         *starr.Value `json:"cutoff"`
+	Languages      []Language   `json:"languages"`
+}
+
+// Language is part of LanguageProfile.
+type Language struct {
+	Allowed  bool         `json:"allowed"`
+	Language *starr.Value `json:"language"`
+}
+
+// Define Base Path for Language Profile calls.
+const languageProfileBP = APIver + "/languageProfile"
```

Comment:
I like this idea, and thought about something similar. Lets move the suffix to a prefix. Makes auto completion a lot handier for me. 
```suggestion
const bpLanguageProfile = path.Join(APIver, "languageProfile")
```

(don't click accept on this suggestion since there are other changes needed)

Reply:
using `path.join` here won't work. oh well.